File: src/backend/api/src/main/java/dev/agasen/ecom/api/core/order/model/OrderDetails.java
package dev.agasen.ecom.api.core.order.model;

import java.util.List;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class OrderDetails {
  private PurchaseOrder purchaseOrder;
  private List<OrderItem> orderItems;
  private Long totalPayment;
}

File: src/frontend/e-commerce-ui/src/app/api/models/order-details.ts
/* tslint:disable */
/* eslint-disable */
import { OrderItem } from '../models/order-item';
import { PurchaseOrder } from '../models/purchase-order';
export interface OrderDetails {
  orderItems?: Array<OrderItem>;
  purchaseOrder?: PurchaseOrder;
  totalPayment?: number;
}
